lib/spack/spack/detection/path.py

```diff
@@ -40,6 +40,12 @@
 
 
 def common_windows_package_paths() -> List[str]:
+    """Get the paths for common package installation location on Windows
+    that are outside the PATH
+    Returns [] on *nix
```

Comment:
Our docs builder doesn't like the `*` (it thinks you are trying to bold something) - I recommend just saying Unix